source/framework/core/inc/TRestAnalysisPlot.h

```diff
@@ -107,6 +107,7 @@ class TRestAnalysisPlot : public TRestMetadata {
 
     struct Panel_Info {
```

Comment:
```suggestion
    struct PanelInfo {
```

I think we should use the same naming convetion used for classes with structs too.